sql/core/src/test/resources/sql-tests/inputs/udf/udf-group-analytics.sql

```diff
@@ -0,0 +1,62 @@
+CREATE OR REPLACE TEMPORARY VIEW testData AS SELECT * FROM VALUES
```

Comment:
This doesn't include a comment like `-- This test file was converted from ...`?

Reply:
@viirya fixed.